products/models.py

```diff
@@ -27,6 +38,13 @@ class Product(TimeStampedModel):
     category = models.ForeignKey(Category, on_delete=models.CASCADE)
     brand = models.ForeignKey(Brand, on_delete=models.CASCADE)
     deleted_at = models.DateTimeField(null=True)
+    objects = ProductManager()
+    allobjects = AllProductManager()
```

Comment:
please rename "allobjects" to "all_abjects" and you can use default manager don't create customer.